examples/kubeconfig/go.mod

```diff
@@ -0,0 +1,69 @@
+module github.com/christensenjairus/multicluster-runtime/examples/kubeconfig
+
+go 1.24.1
+
+replace sigs.k8s.io/multicluster-runtime => ../../
+
+require (
+	github.com/go-logr/logr v1.4.2
+	golang.org/x/sync v0.8.0
+	k8s.io/api v0.32.3
+	k8s.io/client-go v0.32.3
+	sigs.k8s.io/controller-runtime v0.20.4
+	sigs.k8s.io/multicluster-runtime v0.20.0-alpha.5
```

Comment:
does this introduce any new dependencies? If not, I would remove the go.mod file. Makes handling a lot easier (e.g. during tagging).

Reply:
removed the go.mod and go.sum